index.html

```diff
@@ -0,0 +1,231 @@
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+            <a href="facebook.com">
+                <img src="./assets/pic.svg" alt="">
+            </a> 
+  
+             <a href="instagram.com">
+              <img src="./assets/pic1.svg" alt="">
+                
+             </a>
+  
+             <a href="twitter.com">
+             <img src="./assets/pic2.svg" alt="">
+                
+             </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cards__card cards__card--card1 ">
+      <div class="cards__card cards__card--img">
+
+        <img  class="cards__image" src="./assets/user-cover-3.svg" alt="">
+        <div class="cards__lower-box">
+          <h5 class="cards__names">
+                       Julian Jameson
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+            <a href="facebook.com">
+                <img src="./assets/pic.svg" alt="">
+            </a> 
+  
+             <a href="instagram.com">
+              <img src="./assets/pic1.svg" alt="">
+                
+             </a>
+  
+             <a href="twitter.com">
+             <img src="./assets/pic2.svg" alt="">
+                
+             </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cards__card cards__card--card1 ">
+      <div class="cards__card cards__card--img">
+
+        <img  class="cards__image" src="./assets/user-cover-4.svg" alt="">
+        <div class="cards__lower-box">
+          <h5 class="cards__names">
+                       Julian Jameson
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+             <a href="facebook.com">
+              <img src="./assets/pic.svg" alt="">
+          </a> 
+
+           <a href="instagram.com">
+            <img src="./assets/pic1.svg" alt="">
+              
+           </a>
+
+           <a href="twitter.com">
+           <img src="./assets/pic2.svg" alt="">
+              
+           </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
+</div>
+
+
+
+
+<div class="page4">
+
+  <div class="page4__body">
+  <h6 class="heading6 heading6--page4">Newsletter</h6>
+  <h3 class="heading3 heading3--page4" >JOIN US</h3>
+  <p  class="paragraph paragraph--page4" >Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
+  </div>
+  <div class="page4__footer">
+    <input  class="input" type="text" placeholder="Your Email">
+    <button class="input__button"><p class="input__subscribe">Subscribe</p></button>
```

Comment:
Here we have `input__button` class, I expect this to be inside a BLOCK called `input` but looks it is a sibling with one instead.

Reply:
Same with `input__subscribe` does not have any BLOCK with `input`